bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -406,6 +413,17 @@ private void init() throws Exception {
 		personManager = new PersonManager(this);
 		hostAddressManager = new HostAddressManager(this);
 	}
+	
+	/**
+	 * Returns the custom content provider for this case if one exists.
+	 * Otherwise, returns null.
+	 *
+	 * @return The custom content provider for this case if one exists.
+	 *         Otherwise, returns null.
+	 */
+	ContentProvider getContentProvider() {
```

Comment:
Should this return an Optional instead of NULL?

Reply:
@bcarrier , sure.  I will change it to return `Optional<ContentProvider>`